### PR TITLE
amass: add Chinese translation

### PR DESCRIPTION
**Pull request by SignorMercurio**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

 - **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.zh/common/amass.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# amass
+
+> 深度攻击面探测与资产发现工具。
+> 部分子命令如 `amass db` 拥有独立的文档。
```
Comment (blueskyson) [inline pages.zh/common/amass.md:4]: We have a [template](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/subcommand-mention.md) for it.
```suggestion
> 此命令也有关于其子命令的文件，例如：`amass db`.
```

File: pages.zh/common/amass.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+> 更多信息：<https://github.com/OWASP/Amass>。
```
Comment (blueskyson) [inline pages.zh/common/amass.md:5]:
```suggestion
> 更多信息：<https://github.com/OWASP/Amass>.
```

File: pages.zh/common/amass.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+- 查看 Amass 版本:
```
Comment (blueskyson) [inline pages.zh/common/amass.md:7]: Use full-width colon if a sentence is ends with a Chinese word.
```suggestion
- 查看 Amass 版本：
```

File: pages.zh/common/amass.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+`amass -version`
+
+- 展示帮助信息:
```
Comment (blueskyson) [inline pages.zh/common/amass.md:11]:
```suggestion
- 展示帮助信息：
```

File: pages.zh/common/amass.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+`amass -help`
+
+- 展示子命令帮助信息（如 `intel`，`enum` 等）:
```
Comment (blueskyson) [inline pages.zh/common/amass.md:15]:
```suggestion
- 展示子命令帮助信息（如 `intel`、`enum` 等）：
```

File: pages.zh/common/amass.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+`amass -help {{子命令}}`
+
+- 执行 Amass 子命令:
```
Comment (blueskyson) [inline pages.zh/common/amass.md:19]:
```suggestion
- 执行 Amass 子命令：
```

**Review by blueskyson — COMMENTED** [review]
Thanks for contribution. Here are some suggestions.

**Comment by SignorMercurio** [comment]
> Thanks for contribution. Here are some suggestions.

Thank you for your suggestions. I've commited these suggestions, could you please check if it's ok?

**Review by navarroaxel — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]
👍

**Pull request merged**